econsa/copula.py

```diff
@@ -0,0 +1,105 @@
+"""Conditional sampling from Gaussian copula.
+
+This module contains functions to sample random input parameters from a Gaussian copula.
+"""
+import numpy as np
+from scipy.stats import multivariate_normal as multivariate_norm
+from scipy.stats import norm
+
+from econsa.sampling import cond_mvn
+
+
+def cond_gaussian_copula(cov, dependent_ind, given_ind, given_value_u):
```

Comment:
Please add the ability to sample more than one deviate from the conditional distribution by adding an argument with a default value `size=1` and then use that in line 70 to draw multiple deviates.